src/components/layout/navigation/Sorting.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from "react";
+import MenuItem from '@mui/material/MenuItem';
+import Select, {SelectChangeEvent} from '@mui/material/Select';
+import Box from "@mui/material/Box";
+import {Typography} from "@mui/material";
+
+export const Sorting = () => {
+    const [value, setValue] = React.useState('1');
+    const sort = ['Verify', 'Date', 'Popularity', 'Something else']
```

Comment:
if ve gonna use this array in the future, create please, constant file for it. or even better use enum